extensions/sql-migration/src/wizard/sqlSourceConfigurationPage.ts

```diff
@@ -0,0 +1,108 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as azdata from 'azdata';
+import * as os from 'os';
+import { MigrationWizardPage } from '../models/migrationWizardPage';
+import { MigrationStateModel, StateChangeEvent } from '../models/stateMachine';
+import * as constants from '../constants/strings';
+import { createLabelTextComponent, createHeadingTextComponent } from './wizardController';
+
+export class SqlSourceConfigurationPage extends MigrationWizardPage {
+	private _view!: azdata.ModelView;
+
+	constructor(wizard: azdata.window.Wizard, migrationStateModel: MigrationStateModel) {
+		super(wizard, azdata.window.createWizardPage(constants.SOURCE_CONFIGURATION, 'MigrationModePage'), migrationStateModel);
+	}
+
+	protected async registerContent(view: azdata.ModelView): Promise<void> {
+		this._view = view;
+		const form = view.modelBuilder.formContainer()
+			.withFormItems(
+				[
+					await this.createSourceCredentialContainer(),
+				]
+			);
+		await view.initializeModel(form.component());
+	}
+
+	public async onPageEnter(): Promise<void> {
```

Comment:
These are now overriding the base class so it's not going to clear the navigation validation -is that intended?